data-otservbr-global/scripts/actions/quests/adventures_of_galthen/an_idol.lua

```diff
@@ -1,19 +1,20 @@
 local idol = {
```

Comment:
```lua
local anIdolStatue = Action()

function anIdolStatue.onUse(player, item, fromPosition, target, toPosition, isHotkey)
	player:getPosition():sendMagicEffect(CONST_ME_TELEPORT)
	player:teleportTo(Position(32366, 32531, 8))
	player:getPosition():sendMagicEffect(CONST_ME_TELEPORT)
	return true
end

anIdolStatue:position(Position(32398, 32509, 7))
anIdolStatue:register()
```

Reply:
> ```lua
> local anIdolStatue = Action()
> 
> function anIdolStatue.onUse(player, item, fromPosition, target, toPosition, isHotkey)
> 	player:getPosition():sendMagicEffect(CONST_ME_TELEPORT)
> 	player:teleportTo(Position(32366, 32531, 8))
> 	player:getPosition():sendMagicEffect(CONST_ME_TELEPORT)
> 	return true
> end
> 
> anIdolStatue:position(Position(32398, 32509, 7))
> anIdolStatue:register()
> ```

pensei em tirar do for, mas como eu não conheço a mecânica, talvez poderia existir outras "estátuas" e daí teria que ser no for.